lightspeed/modules/proc_create-connection-secrets.adoc

```diff
@@ -139,4 +139,8 @@ secretGenerator:
       disableNameSuffixHash: true
 ....
 
-After you create the bundle secret, you must select the secret when you  xref:create-lightspeed-instance_configuring-lightspeed-onpremise[create and deploy an {LightspeedshortName} instance].
\ No newline at end of file
+After you create the bundle secret, you must select the secret when you  xref:create-lightspeed-instance_configuring-lightspeed-onpremise[create and deploy an {LightspeedshortName} instance].
```

Comment:
Minor nit; remove extra space.
```suggestion
After you create the bundle secret, you must select the secret when you xref:create-lightspeed-instance_configuring-lightspeed-onpremise[create and deploy an {LightspeedshortName} instance].
```

Reply:
Thanks for catching that. Updated the doc